admin-dev/themes/new-theme/js/pages/product/services/attribute-groups.ts

```diff
@@ -22,17 +22,17 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-
+/* eslint-disable max-len */
 import Router from '@components/router';
 
 const router = new Router();
 const {$} = window;
 
-export const getProductAttributeGroups = async (productId) => $.get(router.generate('admin_products_attribute_groups', {
+export const getProductAttributeGroups = async (productId: number): Promise<Array<Record<string, any>>> => $.get(router.generate('admin_products_attribute_groups', {
```

Comment:
Here too if we had some `AttributeGroup` type it would reduce the size of the line and prevent using a disabled comment